src/components/image.ts

```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react';
+import styled from 'styled-components';
+import { compose, layout, position } from 'styled-system';
+import * as SS from 'styled-system';
+
+type ImageType = FC<SS.LayoutProps & SS.PositionProps>;
+
+const Image = styled.img(compose(layout, position)) as ImageType;
+const FullSizeImage = (styled(Image) as unknown) as ImageType;
+
+FullSizeImage.defaultProps = {
+  maxWidth: '100%',
+  maxHeight: '100%'
+};
+
+export { Image, FullSizeImage };
```

Comment:
Note:
When renaming files, if 50% of the file changes, then Git marks it as a "delete + create" instead of a rename.
Since this file was so small, it looks like the (minor) TS conversion went past that 50% mark (8 lines)
I tried hard in this PR to minimize how often this happens.